.storybook/recipes/DataSummaryCard/DataSummaryCard.tsx

```diff
@@ -0,0 +1,91 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './DataSummaryCard.module.css';
+
+import { Card, Heading, Text } from '../../../src';
+
+export interface Props {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Main data text of the card.
+   */
+  data?: string;
+  /**
+   * Text that provides a unit of measurement for the data.
+   */
+  dataUnit?: string;
+  /**
+   * Text to provide more context for the data.
+   */
+
+  description?: string;
+  /**
+   * Title text of the data to represented.
+   */
+  title: string;
+  /**
+   * Off track variant to indicate status.
+   */
+  variant?: 'off-track';
+}
+
+/**
+ * Recipe for a Card component to showcase categorical statistics.
+ */
+export const DataSummaryCard = ({
+  className,
+  data,
+  dataUnit,
+  description,
+  title,
+  variant,
+  ...other
+}: Props) => {
+  const componentClassName = clsx(styles['data-summary-card'], className);
+  return (
+    <Card className={componentClassName} {...other}>
+      <Card.Header className={styles['data-summary-card__header']}>
+        <Heading className={styles['data-summary-card__title']} size="h3">
+          {title}
+        </Heading>
+      </Card.Header>
+      <Card.Body className={styles['data-summary-card__body']}>
+        <Text
```

Comment:
I see that both `data` and `dataUnit` are optional – what does this look like if neither are passed in? Should `data` at least be required?

Reply:
`data`/`dataAmount` should be required yup, will fix